index.html

```diff
@@ -4,6 +4,10 @@
   <title>YOUR NAME HERE</title>
 </head>
 <body>
+  <h1>Kirby Tate </h1>
+  <img src="http://giphy.com/gifs/pDVK1sjLVuMA8">
+
+  </template>
```

Comment:
I was just reading about `template` this weekend. Interesting tag. But not appropriate here! :)